motor_controller:
  ros__parameters:
    enable_priorities: false
    num_motors: 6
    motor0:
      surge: 0.5
      sway: -0.1339
      heave: 0.0
      roll: -2.9193
      pitch: 0.0
      yaw: 3.2051

    motor1:
      surge: 0.5
      sway: 0.1339
      heave: 0.0
      roll: 2.9193
      pitch: 0.0
      yaw: -3.2051

    motor2:
      surge: -0.0706
      sway: 0.3868
      heave: -0.4685
      roll: -1.7816
      pitch: -1.9560
      yaw: 0.0

    motor3:
      surge: -0.0706
      sway: 0.3203
      heave: 0.4
      roll: 1.7816
      pitch: -1.9560
      yaw: 0.0

    motor4:
      surge: -0.0706
      sway: -0.3203
      heave: 0.4
      roll: -1.7816
      pitch: -1.9560
      yaw: 0.0

    motor5:
      surge: -0.0706
      sway: -0.3868
      heave: -0.4685
      roll: 1.7816
      pitch: -1.9560
      yaw: 0.0